.vscode/settings.json

```diff
@@ -8,4 +8,12 @@
     "spellright.ignoreRegExpsByClass": {
         "markdown": ["/:\\w+:/g"]
     },
+    "spellright.language": [
+      "en"
+    ],
+    "spellright.documentTypes": [
+      "latex",
```

Comment:
We have LaTeX documents?

Reply:
latex (with plaintext and markdown) automatically got inserted into the workspace settings when I selected English as the spellcheck language.